itests/org.openhab.binding.mqtt.homie.tests/src/main/java/org/openhab/binding/mqtt/HomieImplementationTest.java

```diff
@@ -147,7 +147,7 @@ public void setUp() throws InterruptedException, ExecutionException, TimeoutExce
         futures.add(embeddedConnection.publish(propertyTestTopic + "/$datatype", "boolean".getBytes()));
 
         registeredTopics = futures.size();
-        CompletableFuture.allOf(futures.toArray(new CompletableFuture[0])).get(1000, TimeUnit.MILLISECONDS);
+        CompletableFuture.allOf(futures.toArray(new CompletableFuture[0])).get(1500, TimeUnit.MILLISECONDS);
```

Comment:
This change does not seem to be related to this PR - if you want to change this code, please do it in a separate PR for better traceability.

Reply:
I can remove that. However, it doesn't compile without that for me.